app/submitter/submitter.py

```diff
@@ -56,7 +55,7 @@ def send_message(
         # DEFAULT_RETRY is not idempotent.
         # However, this behaviour was deemed acceptable for our use case.
```

Comment:
Can remove the comment now to avoid confusion 👍

Reply:
Removed, thanks.